### PR TITLE
Dashboard form enhancement

### PR DESCRIPTION
**Pull request by NetanelK**
Dashboard form enhancement for #106 

- [X] Let all the Textboxes have a useful placeholder such as an example or description
- [X] Change Label name for Ended as Ends on

- [X] Expect Title all other input boxes(Including started on and ends on) should be hidden and should have an EDIT button/text that will act as a trigger to display the input box and once input has been provided with data, input box needs to be hidden and entered input must be displayed on the side of the edit button (Reach out design team for suggestions)

- [X] Move noteworthy task below Title

- [X] Change default value for following Completion Award: Dinero:1000,Loss Rate:Dinero:100 and if its a noteworthy task the dafult value for Completion Award: Dinero: should be 2500

- [X] Let date have a default value of 7 days and display the date such as 7 days remaining

- [X] Default value for Status should be active

https://user-images.githubusercontent.com/20744671/141986740-b671b4ff-3d41-4187-aa87-2d99390573ac.mp4

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -42,8 +42,12 @@ body {
 .inputBox label {
   font-size: 1rem;
   font-weight: 700;
+  width: 100%;
+  text-align: left;
 }
 
+
+
```
Comment (Pavangbhat) [inline main.css:50]: Can you please remove an extra line here

File: main.css [hunk 2]
```diff
@@ -200,3 +204,15 @@ footer {
   background-color: #1d1283;
   border: none;
 }
+
+#remainingDays {
+  /* all:unset; */
+  float: left;
+  margin-block: -.5em .75em;
+  padding-left: .5em;
+  font-size: 14px;
```
Comment (Pavangbhat) [inline main.css:213]: If possible can we use em her for consistency

File: task/create.html [hunk 3]
```diff
@@ -17,36 +17,48 @@
             class="input"
             type="text"
             name="title"
-            placeholder="Title"
+            placeholder="Your task name"
```
Comment (Pavangbhat) [inline task/create.html:20]: Can we have an example here such as "eg:Testing setup for dashboard"

File: task/create.html [hunk 4]
```diff
@@ -17,36 +17,48 @@
...
+        <div class="inputBox">
+          <label for="purpose" class="editable">Purpose:</label>
           <textarea
             type="text"
-            class="textInput"
+            class="textInput notEditing"
             id="purpose"
             name="purpose"
-            placeholder="Purpose"
+            placeholder="What is your task about?"
```
Comment (Pavangbhat) [inline task/create.html:39]: Can it have an small example "eg:To test behaviour of code"

File: task/create.html [hunk 5]
```diff
@@ -58,7 +70,7 @@
               <label for="group">Group</label>
               <input
                 required
-                class="input"
+                class="input "
```
Comment (Pavangbhat) [inline task/create.html:73]: Can we please remove empty spaces here and at line 61

File: task/create.html [hunk 6]
```diff
@@ -69,121 +81,126 @@
...
         </div>
-        <div class="inputBox">
-          <div class="checkBox">
-            <label for="isNoteworthy">Is noteworthy:</label>
-            <span
-              ><input type="checkbox" id="isNoteworthy" name="isNoteworthy"
-            /></span>
-          </div>
-        </div>
+
```
Comment (Pavangbhat) [inline task/create.html:203]: Can we please remove this empty line

**Review by Pavangbhat — CHANGES_REQUESTED** [review]
Hi @NetanelK have requested a few changes can you please work on that
And please run **npm fix** before committing and pushing your changes 
Notice your Github actions are failing

File: main.css (hunk 2 above)
Comment (NetanelK) [reply main.css:213]: I did use em

**Comment by NetanelK** [comment]
Add fixes to #109, and requested fixes.

File: task/create.html [hunk 7]
```diff
@@ -17,28 +17,39 @@
...
+            placeholder="eg: Testing setup for dashboard"
             id="title"
           />
         </div>
         <div class="inputBox">
-          <label for="purpose">Purpose:</label>
+          <div class="checkBox">
+            <label for="isNoteworthy">Is noteworthy:</label>
+            <span
+              ><input type="checkbox" id="isNoteworthy" name="isNoteworthy"
```
Comment (Pavangbhat) [inline task/create.html:28]: Can you please format your code using prettier(need to run npm install) and run npm fix

**Comment by Pavangbhat** [comment]
I have left few comments please have a look on that.
Other than that every things good to me

File: task/create.html (hunk 7 above)
Comment (NetanelK) [reply task/create.html:28]: I did run npm fix, and all checks have passed

Comment (Pavangbhat) [reply task/create.html:28]: Yes that's right, but linting does not format your code but it checks for a few rules 
So please format your code and run **npm fix** as well
Thank you :smiley:

Comment (NetanelK) [reply task/create.html:28]: When I run **npm run fix** nothing changed

Comment (Pavangbhat) [reply task/create.html:28]: Got it :+1: 
We have a linting issue will fix that

**Pull request merged**